crates/agglayer-certificate-orchestrator/src/lib.rs

```diff
@@ -197,22 +197,24 @@ where
     ///             Ok(CertifierOutput {
     ///                 new_state: LocalNetworkStateData::default().into(),
     ///                 network: NetworkId::new(0),
-    ///                 certificate: Certificate::new_for_test(network_id, height),
-    ///                 height: 0,
+    ///                 certificate: Certificate::new_for_test(network_id,
+    /// height),                 height: 0,
```

Comment:
The formatter is trying to format the comment in a way that's totally not appropriate for a code block. Is it possible to turn it off locally just for the comment?